llvm/test/CodeGen/SystemZ/machine-ver-kill-memop.mir

```diff
@@ -0,0 +1,52 @@
+# RUN: llc -o /dev/null %s -mtriple=s390x-linux-gnu -mcpu=z16 \
+# RUN:   -verify-machineinstrs -run-pass=register-coalescer
+
+# Test that the MemoryOperand of the produced KILL instruction is removed
+# and as a result the machine verifier succeeds.
+
+--- |
+  define void @fun(ptr %Src, ptr %Dst, ptr %Src2) {
+    ret void
+  }
+...
+---
+name:            fun
+alignment:       16
+tracksRegLiveness: true
+noPhis:          true
+isSSA:           false
+noVRegs:         false
+hasFakeUses:     false
+registers:
+  - { id: 0, class: grx32bit }
+  - { id: 1, class: grx32bit }
+  - { id: 2, class: grx32bit }
+  - { id: 3, class: addr64bit }
+  - { id: 4, class: gr64bit }
+  - { id: 5, class: grx32bit }
+  - { id: 6, class: grx32bit }
+  - { id: 7, class: grx32bit }
+  - { id: 8, class: addr64bit }
+liveins:
+  - { reg: '$r2d', virtual-reg: '%3' }
+frameInfo:
+  maxAlignment:    1
+machineFunctionInfo: {}
```

Comment:
```suggestion
tracksRegLiveness: true
```

Reply:
actually, that line is already present further up...

Reply:
This is to delete everything else, not add the line